ci-operator/step-registry/cucushift/installer/rehearse/aws/ipi/ovn/hypershift/private/guest/cilium/cucushift-installer-rehearse-aws-ipi-ovn-hypershift-private-guest-cilium-workflow.metadata.json

```diff
@@ -0,0 +1,15 @@
+{
+	"path": "cucushift/installer/rehearse/aws/ipi/ovn/hypershift/private/guest/cilium/cucushift-installer-rehearse-aws-ipi-ovn-hypershift-private-guest-cilium-workflow.yaml",
+	"owners": {
+		"approvers": [
```

Comment:
I guess the owner list should be updated to include hypershift team members.

Reply:
Right, added https://github.com/openshift/release/pull/65905/commits/54ecb89070fc97db46573805e7b41d8a6af87827